cmake/modules/packaging.cmake

```diff
@@ -137,7 +137,7 @@ macro(CREATE_PYTHON_EXE EXE_NAME MAIN_MODULE)
         "f.close()\n")
 
     # Run Python to generate the zipped file
-    execute_process(COMMAND python "${BUILD_DIR_MAIN}/do_zip.py"
+    execute_process(COMMAND ${PYTHON_EXECUTABLE} "${BUILD_DIR_MAIN}/do_zip.py"
```

Comment:
Hi @asgardkm, I think that this file has been added mistakenly.  These macros are now located in opae-libs/cmake/modules/OPAEPackaging.cmake.  Can you take a look at that file and update it, removing this one from the pull request?